app/components/more-details.jsx

```diff
@@ -0,0 +1,92 @@
+// https://github.com/EnCiv/civil-pursuit/issues/89
+
+'use strict'
+import React, { useState } from 'react'
+import Column from './util/column'
+import Submit from './util/submit'
+import { createUseStyles } from 'react-jss'
+import { JsonForms } from '@jsonforms/react'
+import { vanillaCells, vanillaRenderers } from '@jsonforms/vanilla-renderers'
+
+const MoreDetails = props => {
+  const { onDone = () => {}, initialData = {} } = props
+  const [data, setData] = useState(initialData)
+  const classes = useStyles(props)
+
+  const schema = {
```

Comment:
schema and uischema should be in props, the values here can be moved to the stories. Also, per the definition in the issue, initialData should be called details.